### PR TITLE
implemented rendering of images, is slow

### PR DESCRIPTION
**Pull request by joshinils**
i fear the path will not be read correctly. for me it renders the image when holding down spacebar, otherwise i have issues displaying the image, it only shows some pixels while rotating or all when translating with `r`

### CODE REVIEW

File: engine/olcPixelGameEngine.h [hunk 1]
```diff
@@ -854,8 +854,8 @@ namespace olc
 
 	Pixel Sprite::Sample(float x, float y)
 	{
-		int32_t sx = (int32_t)(x * (float)width);
-		int32_t sy = (int32_t)(y * (float)height);
+		uint32_t sx = (uint32_t)(x * (float)width);
+		uint32_t sy = (uint32_t)(y * (float)height);
 		return GetPixel(sx, sy);
```
Comment (philsupertramp) [inline engine/olcPixelGameEngine.h]: We're referencing a not existing object within [`return Pixel(0, 0, 0, 0);`](https://github.com/philsupertramp/wirdnix/blob/a87884755142f1aeb53d5a10fa3abfa2a1ae1297/engine/olcPixelGameEngine.h#L834)

Comment (philsupertramp) [reply engine/olcPixelGameEngine.h]: change ` return Pixel(0, 0, 0, 0)` to `return olc::BLANK;`

File: engine/olcPGEX_Graphics3D.h [hunk 2]
```diff
@@ -76,11 +76,10 @@
 namespace olc
```
[the remark this answers is not in the document]
Comment (joshinils) [reply engine/olcPGEX_Graphics3D.h]: k

File: engine/olcPixelGameEngine.h (hunk 1 above)
Comment (joshinils) [reply engine/olcPixelGameEngine.h]: is the change to explicitly unsigned the killer for you? or was it something else that changed for better performance on your end?

File: engine/olcPGEX_Graphics3D.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshinils) [reply engine/olcPGEX_Graphics3D.h]: idk how to insert a return statement there, the function  seems to convert a string to a wide string. if that is even necessary somewhere for us

**Pull request merged**